assets/chezmoi.io/docs/reference/special-directories/chezmoitemplates.md

```diff
@@ -1,12 +1,13 @@
-# `.chezmoitemplates`
+# `.chezmoitemplates/`
 
-If a directory called `.chezmoitemplates` exists, then all files in this
-directory are available as templates with a name equal to the relative path
-to the `.chezmoitemplates` directory.
+If a directory called `.chezmoitemplates/` exists in the root of the source
```

Comment:
@twpayne I am not sure that my change here is *correct* (are `.chezmoitemplates/` directories allowed anywhere in the source tree or not), but I can't find examples of non-root `.chezmoitemplates/` in a GitHub search.

Reply:
`.chezmoitemplates` are allowed anywhere, but I don't think anyone uses this either.